pkg/util/crds.go

```diff
@@ -0,0 +1,268 @@
+package util
+
+import (
+	core "k8s.io/api/core/v1"
+	kerr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	api_v1beta1 "stash.appscode.dev/stash/apis/stash/v1beta1"
+	cs "stash.appscode.dev/stash/client/clientset/versioned"
+	"stash.appscode.dev/stash/pkg/docker"
+)
+
+// EnsureDefaultFunctions creates "update-status", "pvc-backup" and "pvc-restore" Functions if they are not already present
+func EnsureDefaultFunctions(stashClient cs.Interface, registry, imageTag string) error {
+	image := docker.Docker{
+		Registry: registry,
+		Image:    docker.ImageStash,
+		Tag:      imageTag,
+	}
+
+	defaultFunctions := []*api_v1beta1.Function{
+		updateStatusFunction(image),
+		pvcBackupFunction(image),
+		pvcRestoreFunction(image),
+	}
+
+	for _, fn := range defaultFunctions {
+		_, err := stashClient.StashV1beta1().Functions().Create(fn)
```

Comment:
Should we be using CreateOrPatch methods?

Reply:
Fixed